src/compute/src/sink/mod.rs

```diff
@@ -10,6 +10,7 @@
 mod avro_ocf;
 mod kafka;
 mod metrics;
+mod persist_sink;
```

Comment:
Eventually I think we'll want to put these over in STORAGE, but for the time being we cannot (TAIL being a SINK complicates that). Nothing to do at the moment here!